From dd1fdc841d069dbd4e284f430a88af79de951124 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Thu, 26 Sep 2019 17:54:00 +0800
Subject: [PATCH] python-lttngust/Makefile.am: Add --install-lib to setup.py

Otherwise it may install to /usr/lib, but should be /usr/lib64 when cross
building.

Upstream-Status: Submitted [https://github.com/lttng/lttng-ust/pull/59]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>

---
 src/python-lttngust/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/python-lttngust/Makefile.am b/src/python-lttngust/Makefile.am
index f95482d..f76d95b 100644
--- a/src/python-lttngust/Makefile.am
+++ b/src/python-lttngust/Makefile.am
@@ -32,7 +32,7 @@ install-exec-local: build-python-bindings.stamp
 	if [ "$(DESTDIR)" != "" ]; then \
 		opts="$$opts --root=$(DESTDIR)"; \
 	fi; \
-	$(PYTHON) $(builddir)/setup.py install $$opts;
+	$(PYTHON) $(builddir)/setup.py install $$opts --install-lib=$(pythondir);
 
 clean-local:
 	rm -rf $(builddir)/build
